app/views/teaching_events/about_git_events.html.erb

```diff
@@ -10,26 +10,30 @@
   </section>
 
   <section class="col col-720">
-    <p>Whether you're ready to start your career in teaching or just curious, we can answer your questions at a Get Into Teaching event.</p>
+    <p>We understand everyone’s journey into the classroom is different and your questions may be unique to your circumstances. But whether you're ready to start your career in teaching, or just curious, we can answer your queries at a Get Into Teaching event.</p>
+
+    <p>These relaxed and informative events provide a one-stop shop of information and support. They'll give you the opportunity to get expert advice and talk to teachers and local training providers.</p>
 
-    <p>These relaxed and informative events give you the opportunity to get expert advice and talk to teachers and local training providers.</p>
   </section>
 
   <section class="col col-720">
     <h3 class="heading-m">At our Get Into Teaching events</h3>
 
-    <p>Drop in any time during an event for presentations, one-to-one chats or to meet others like you who are thinking of stepping into the classroom.</p>
+    <p>Our advisers can help you understand the qualifications you need and the range of different ways you can train.</p>
+    
```

Comment:
Looks like there is a linting issue on this line?

Reply:
I've deleted this bit now so hopefully it's fixed it.